### PR TITLE
feat: remove faq, general, motive, information, vision

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: site/src/modules/integration-documentation/index.ts [hunk 1]
```diff
@@ -12,17 +12,17 @@ export const IntegrationDocumentationRoutes = [
             {
                 path: "",
                 component: IntegrationDocumentation,
-                name: "integration-documentatie"
+                name: "integratie-documentatie"
```
Comment (jvandaal) [inline site/src/modules/integration-documentation/index.ts:15]: Why the change?

Comment (pgallik) [reply site/src/modules/integration-documentation/index.ts:15]: mix of english and nl
